### PR TITLE
[WIP][NOT READY]

### PR DESCRIPTION
**Pull request by Shnoulle**
Example of usage of https://github.com/LimeSurvey/LimeSurvey/pull/1121

### CODE REVIEW

File: views/index.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="row">
     <div class="col-lg-12 content-right">
 
-        <h3>Mass action</h3>
+        <h3>{{ gT("Mass action",'html',null,'pluginMessagesMassAction') }}</h3>
```
Comment (olleharstedt) [inline views/index.twig:4]: One thing I mildly dislike about this is the number of arguments in the `gT` function call. Could we wrap it in `$plugin->gT()` and make `pluginMessagesMassAction` a default search folder? Or, that `$plugin->gT()` reads a setup file for where to look for plugin-specific translations.

Comment (Shnoulle) [reply views/index.twig:4]: But, purpose is to use it in twig (and other files), without $plugin var :).

And if you want' to use another plugin translation : no way … with $plugin->gT

Comment (Shnoulle) [reply views/index.twig:4]: See this usage too : https://github.com/LimeSurvey/LimeSurvey/pull/1121#issuecomment-421406017 it's out of plugin : directly in theme :).

Comment (Shnoulle) [reply views/index.twig:4]: And more : this don't broke usage of $this->gT , or $plugin->gT : they work like before. This add a way to use ir out of existing Class : https://github.com/LimeSurvey/LimeSurvey/pull/1121/files#diff-de5ed4c534457ee3003a9ae2352e4eceR322

Comment (olleharstedt) [reply views/index.twig:4]: OK.

**Pull request closed without merging**